packages/flutter/lib/src/material/navigation_bar.dart

```diff
@@ -300,12 +308,6 @@ class NavigationDestination extends StatelessWidget {
   /// would use a size of 24.0 and [ColorScheme.onSurface].
   final Widget? selectedIcon;
 
-  /// The color of the [indicatorShape] when this destination is selected.
```

Comment:
AFAICT these properties only landed yesterday - https://github.com/flutter/flutter/pull/116992

So it should be OK to remove them here.

Reply:
Yes, it was merged around 15 hours ago., that's the reason I removed them here and moved up the widget itself (`NavigationBar`) to match the `NavigationRail` API and expanded the PR to cover `NavigationDrawer` for one consistent update.

Reply:
Please let me know if there is any action needed.